app/decorators/guide_decorator.rb

```diff
@@ -17,6 +17,10 @@ def content
     fail 'GuideDecorator subclasses must implement content'
   end
 
+  def tags
```

Comment:
We should only consider exposing the object’s tags if we need to use them in our views.